src/sage/rings/function_field/jacobian_base.py

```diff
@@ -651,6 +654,26 @@ def __call__(self, x):
             True
             sage: J_hess(q) == p
             True
+
+        If ``x`` is an effective divisor, it is checked that the degree
+        is equal to the degree of the base divisor. See :issue:38623.
```

Comment:
```suggestion
        is equal to the degree of the base divisor. See :issue:`38623`.
```

although tests aren't rendered anyway.

Reply:
Thanks. Now TESTS blocks are rendered in doc previews.

Reply:
You are right. The TESTS blocks in a hidden method still do not appear in doc preview.

Reply:
We may include them as well in doc preview for PRs. What do you think?

Reply:
I find sometimes the `TESTS::` are rendered, and sometimes they aren't. I've no idea what's the rule. Either way it doesn't seem to matter too much.

(I think they're supposed to be hidden.)

Reply:
> We may include them as well in doc preview for PRs. What do you think?

I think it would be nice to have the `TESTS::` block included in doc preview for PRs, but I worry about the doc preview for PR not being a preview of what the docs will actually look like in release. Specifically, I don't want to ever have a situation where the doc preview looked fine but the release docs have some issue.

Reply:
Before the recent #38449, TESTS blocks did not appear in any docs at all.

#38449 made TESTS blocks appear in doc previews for PRs, for checking rendering of (newly added) TESTS blocks. In released docs, TESTS blocks do not appear as before (since TESTS are not for users).

In this PR, a TESTS block was added to a method `__call__`. Docstrings (including TESTS blocks) of special methods (with underlines) do not appear in docs at all.  

So what I asked above is essentially whether to include docstrings for special methods (like `__call__`) should be included in doc previews for PRs or not. I think this has certain merits and demerits as well. 

Certainly it is not a simple issue to be discussed in this thread. Nonetheless thanks for your opinions.